hackernews.el

```diff
@@ -507,7 +513,7 @@ rendered at the end of the hackernews buffer."
                                   (max 0 (min (- (length ids) offset)
                                               (if n
                                                   (prefix-numeric-value n)
-                                                hackernews-items-per-page)))
+                                                (hackernews--calculate-story-count))))
```

Comment:
A more Elisp-y way of writing this would have been
```el
(cond (n (prefix-numeric-value n))
      (hackernews-items-per-page)
      (t (window-text-height)))
```